SpringProject/src/main/java/com/gradproject2019/users/controller/UserController.java

```diff
@@ -43,4 +43,10 @@
         UserResponseDto userResponseDto = userService.findUserById(userId, token);
         return ResponseEntity.ok(userResponseDto);
     }
-}
+
+    @GetMapping(path = "/users/employee/{id}") //managers user ID
```

Comment:
The idea of having a fixed parameter (i.e. {id}) in a path like this is that it uniquely identifies some kind of entity. In this case, what you are doing really is filtering all the users by their manager's id. 

In this path, it looks as if that id refers to a type of a `user` called an `employee` that is identified by an {id}. I suggest having this as a query param (i.e. `?manager_id={id}) of the base list users endpoint, that does the same thing. Having a comment to explain this doesn't help anyone using your API. 

This may seem like a minor point, but in an Application Programming _Interface_ the endpoints are all your clients see of your app - getting them right is the most important part of being a good back end dev.